.travis.yml

```diff
@@ -15,7 +16,7 @@ node_js: '10.15' # equivalent to node 10.15.x
 cache:
   directories:
     - node_modules
-#    - $HOME/.npm
+    - /home/travis/.nvm/versions/node/v10.15.3/lib/node_modules/
```

Comment:
This looks brittle! Is there a more robust way to do it, using something closer to that `$HOME` variable?

Reply:
Agree with @chadoh is there a better "timeproof" way? What kind of issues did you find with the previous `$HOME` variable @topocount ?

Reply:
I'll address each of these changes separately:

# Removing the `$HOME/.npm` line
This was commented out anyway, but I read in the TravisCI docs that [`.npm` is now cached by default](https://docs.travis-ci.com/user/caching/#npm-cache) so I figured we should remove it since this is officially documented

# caching global node_modules
This is definitely not pretty. I thought it'd be a good idea beause it would speed things up in terms of installing the aragonCLI globally for use with deployments. However, I've observed that it actually slows down the cache-warming step substantially (adding 3-4 minutes) so I'll go ahead and remove this line